restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java

```diff
@@ -340,6 +348,80 @@ public void searchDoc(
                 asyncResponse, ErrorHandler.EXCEPTION_TO_RESPONSE));
   }
 
+  @POST
+  @ManagedAsync
+  @ApiOperation(
+      value =
+          "Execute a built-in function (e.g. $push and $pop) against this document. Performance may vary.")
```

Comment:
Not sure how terminology is used elsewhere, but wouldn't operation be against a value within document or documents? Value would be the whole document only if document consists of Array value.